app/views/two_factor_authentication/totp_verification/show.html.slim

```diff
@@ -12,7 +12,7 @@ p.mt-tiny.mb0#code-instructs = t('instructions.2fa.totp_intro_html', \
   .col-12.sm-col-5.mb4.sm-mb0.sm-mr-20p.inline-block
     = text_field_tag :code, '', required: true, autofocus: true,
       pattern: '[0-9]*', class: 'col-12 field monospace mfa',
-      'aria-describedby': 'code-instructs'
+      'aria-describedby': 'code-instructs', maxlength: 6
```

Comment:
I think the "totp" is the recovery code, which is 16 digits.  https://github.com/Houdini/two_factor_authentication/blob/73815c0c5132c01d3ff8b49b198e82d8b0664d9a/lib/two_factor_authentication/models/two_factor_authenticatable.rb#L87 (can be accessed with `Figaro.env.recovery_code_length`